src/ensembl/src/shared/components/instant-download/instant-download-gene/InstantDownloadGene.scss

```diff
@@ -0,0 +1,81 @@
+@import 'src/styles/common';
+
+.checkboxGrid {
+  margin-top: 0.5em;
+}
+
+.label {
+  margin-bottom: 0.5em;
+  font-weight: $light;
+  word-break: break-all;
+
+  .featureId {
+    margin-left: 0.5em;
+    font-weight: $bold;
+  }
+}
+
+.transcriptVis {
+  margin-top: 18px;
+  margin-bottom: 9px;
+}
```

Comment:
Could you please make sure that all the classes in this file are actually used in the layout? This class definitely is not.

Reply:
✅